src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -189,6 +192,28 @@ struct ObserveCoefficientsTrigger : db::SimpleTag {
   }
 };
 
+struct InitialPositionAndVelocity : db::SimpleTag {
+  using type = std::array<tnsr::I<double, 3, Frame::Inertial>, 2>;
+  using option_tags =
+      tmpl::list<domain::OptionTags::DomainCreator<3>,
+                 OptionTags::ExcisionSphere, ::OptionTags::InitialTime>;
+  static constexpr bool pass_metavariables = false;
+  static type create_from_options(
+      const std::unique_ptr<::DomainCreator<3>>& domain_creator,
+      const std::string& excision_sphere_name, const double initial_time) {
+    const auto initial_fot = domain_creator->functions_of_time();
+    const auto domain = domain_creator->create_domain();
+    const auto& excision_sphere =
+        domain.excision_spheres().at(excision_sphere_name);
+    ASSERT(excision_sphere.is_time_dependent(),
+           "excision_sphere not time dependent");
+    const auto& maps = excision_sphere.moving_mesh_grid_to_inertial_map();
+    const auto mapped_tuple = maps.coords_frame_velocity_jacobians(
+        excision_sphere.center(), initial_time, initial_fot);
+    return {std::get<0>(mapped_tuple), std::get<3>(mapped_tuple)};
```

Comment:
At `t = 0` I believe we assume that the grid frame and the inertial frame are the same, so we don't need to call the time dependent maps, you can just use the grid centers and copy them into an inertial tensor. Does the curved scalar wave system assume something different?

Reply:
There are a few time dependent domain creators like `RotationAboutZAxis` or `SphericalCompression` allow you to set `InitialRotationAngle` or `InitialValue` for the compression factor. 

In most cases the grid and inertial frame should coincide at t=0, but given that this is not enforced I would prefer to initialize it like this which does not carry any computational cost.

Reply:
Ok that's fine. In that case, could you add a comment above the `domain_creator->functions_of_time()` to say that we don't care about expiration times here since we're evaluating at the initial time?

Reply:
done and also added a sentence for the docs. thanks for the review!